models/pawn.py

```diff
@@ -15,6 +15,25 @@ def __init__(self, row, col, color, img, house):
     def __str__(self):
         return PAWN
 
+    # verifica se eh um movimento especial do peao (andar duas casas)
+    # se o movimento tiver mais de uma casa de distancia, eh um movimento especial
+    def is_special_move(self, current_pos, desired_pos):
+        return (abs(current_pos[1] - desired_pos[1]) != 1)
+
+    #verifica se eh um movimento de captura
+    def is_capture_move(self, current_pos, desired_pos):
+        return (current_pos[0] != desired_pos[0])
```

Comment:
Esqueci de comentar aqui. É um movimento de captura se o peão se movimentar lateralmente. Ou seja, se o valor da coluna dele for alterado. 
pos=(coluna, linha), entao pos[0] é o valor da coluna.